localisation/ko/docs/03-java.md

```diff
@@ -0,0 +1,38 @@
+# 03: Python에서 Java로 마이그레이션
+
+## 시나리오
+
+Contoso는 다양한 야외 활동 제품을 판매하는 회사입니다. Contoso의 마케팅 부서는 기존 고객과 잠재 고객에게 제품을 홍보하기 위한 마이크로 소셜 미디어 웹사이트를 런칭하고자 합니다.
+
+Python 개발자가 회사를 떠났기 때문에, 이해관계자들은 기존 Python 백엔드 API 앱을 Spring Boot를 사용하여 Java로 마이그레이션하도록 요청했습니다.
+
+이제 Java 개발자로서, 기존 FastAPI 앱을 Spring Boot로 마이그레이션해야 합니다. 참고로 Python과 FastAPI에 대한 지식은 매우 적습니다.
+
+## 전제 조건
+
+준비를 위해 [README](../README.md) 문서를 참조하세요.
+
+## 시작하기
+
+- [GitHub Copilot 에이전트 모드 확인](#github-copilot-에이전트-모드-확인)
+- [커스텀 지시사항 준비](#커스텀-지시사항-준비)
+- [Spring Boot 프로젝트 준비](#spring-boot-프로젝트-준비)
+- [FastAPI API 앱 마이그레이션](#fastapi-api-앱-마이그레이션)
+
+### GitHub Copilot 에이전트 모드 확인
+
+1. GitHub Codespace 또는 VS Code 상단의 GitHub Copilot 아이콘을 클릭하고 GitHub Copilot 창을 여세요.
+
+   ![Open GitHub Copilot Chat](./images/setup-02.png)
+
+1. 로그인 또는 가입을 요청받으면 진행하세요. 무료입니다.
+1. GitHub Copilot 에이전트 모드를 사용하고 있는지 확인하세요.
+
+   ![GitHub Copilot Agent Mode](./images/setup-03.png)
```

Comment:
This image path should point to the original one, which is `../../../docs/images/setup-02.png`.